src/sagas/fetch/index.js

```diff
@@ -18,4 +18,24 @@ export function* fetchSaga(
   return null
 }
 
+export function* fetchAnythingSaga(action: Object): Generator<any, any, any> {
+  try {
+    yield put(actions.clearError(action.name))
+    yield put(actions.sendingRequest(action.name, true))
+    const response = yield call(get, action.query, action.args)
+    if (action.callback) {
+      const callbacks = action.callback(response, action.args)
+      while (callbacks.length) {
```

Comment:
well, it's not possible :) generators can be called only from generators, `map` is iterator but not the generator, so the `yield` won't work with `map`